source_code/main_mcu/src/COMMS/comms_hid_defines.h

```diff
@@ -76,6 +76,7 @@
 #define HID_CMD_GET_FILE_DATA_ID    0x0023
 #define HID_CMD_IM_LOCKED           0x0024
 #define HID_CMD_IM_UNLOCKED         0x0025
+#define HID_CMD_ID_STORE_TOTP_CRED  0x0026
```

Comment:
can you make the PR to update https://github.com/mooltipass/minible/blob/gh-pages/protocol.md ?

Reply:
:+1: